broadcast/broadcast.go

```diff
@@ -32,12 +36,44 @@ func NewCompositeBroadcaster(strategy Strategy, factories ...BroadcastFactory) B
 	}
 }
 
-func (c *compositeBroadcaster) BestQuote(ctx context.Context, feeCategory, feeType string) error {
-	executionFuncs := make([]func(context.Context) error, len(c.broadcasters))
+func (c *compositeBroadcaster) QueryTransaction(ctx context.Context, txID string) (*models.QueryTxResponse, error) {
```

Comment:
Actually unit tests for the compositeBroadcaster would be a good idea.
And actually those unit tests could use mocked or fake implementation of broadcasters

Reply:
As I want to commit this chunk I created subtask for unit testing strategy and compositeBroadcaster.

Reply:
BUX-153